src/test/java/BoardTest.java

```diff
@@ -243,4 +243,131 @@ public void testMovePawnsIllegally() throws InvalidPositionException
 		/*16 b/c there are 16 pawns that get moved illegally*/
 		assertEquals(numErrors, 16);
 	}
+	
+	//Moves pawns forward to test functionality of moving pieces to valid locations. 
```

Comment:
- [ ] This comment makes me think that the test only focuses on moving pawns, but unless I'm mistaken, it also moves other pieces around, right?